src/main/java/org/opensearch/timeseries/settings/TimeSeriesSettings.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.timeseries.settings;
+
+public class TimeSeriesSettings {
+
+    // ======================================
+    // Model parameters
+    // ======================================
+    public static final int DEFAULT_SHINGLE_SIZE = 8;
+
+    // max shingle size we have seen from external users
+    // the larger shingle size, the harder to fill in a complete shingle
+    public static final int MAX_SHINGLE_SIZE = 60;
```

Comment:
Could this number be potentially different between forecasting and AD?

Reply:
So far we can keep it the same. Can change when needed.

Reply:
Sounds good